File: CS/ThemingDescendant/App_Code/CustomLayoutDesigner.cs
using System.ComponentModel;
using DevExpress.Web;
using DevExpress.Web.Design;

namespace CustomLayout {
    [ToolboxItem(false)]
    public class CustomLayoutDesigner : ASPxFormLayoutDesigner {
        public override void Initialize(IComponent component) {
            base.Initialize(component);
            RegisterTagPrefix(typeof(ASPxFormLayout));
        }
    }
}

File: VB/ThemingDescendant/App_Code/CustomLayoutDesigner.vb
Imports System.ComponentModel
Imports DevExpress.Web
Imports DevExpress.Web.Design

Namespace CustomLayout
    <ToolboxItem(False)> _
    Public Class CustomLayoutDesigner
        Inherits ASPxFormLayoutDesigner

        Public Overrides Sub Initialize(ByVal component As IComponent)
            MyBase.Initialize(component)
            RegisterTagPrefix(GetType(ASPxFormLayout))
        End Sub
    End Class
End Namespace